packages/sdk-middleware-http/src/http.js

```diff
@@ -112,9 +112,10 @@ export default function createHttpMiddleware({
             res.json().then((result: Object) => {
```

Comment:
Shouldn't we simply check the method if it's `HEAD` and return without parsing the response body? 🤔